src/c4/test/tstifpstream.cc

```diff
@@ -0,0 +1,161 @@
+//----------------------------------*-C++-*-----------------------------------//
+/*!
+ * \file   c4/test/tstifpstream.cc
+ * \author Mathew Cleveland
+ * \date   Wed Apr 28 09:31:51 2010
+ * \brief  Tests ifpstream class for paralle read
+ * \note   Copyright (C) 2016-2020 Triad National Security, LLC.
+ *         All rights reserved. */
+//----------------------------------------------------------------------------//
+
+#include "c4/ParallelUnitTest.hh"
+#include "c4/ifpstream.hh"
+#include "c4/ofpstream.hh"
+#include "ds++/Release.hh"
+#include <sstream>
+
+using namespace std;
+using namespace rtt_dsxx;
+using namespace rtt_c4;
+
+//----------------------------------------------------------------------------//
+// TESTS
```

Comment:
It might be clearer to have this header be something like "HELPER FUNCTIONS," since the write_stream_* functions directly below aren't the actual ifpstream tests! It's not a big deal, though.

Reply:
fixed